### ISSUE TITLE
Unable to build all docker containers

### ISSUE THREAD

**Issue by Ollrogge** [fuzzware-fuzzer/hoedur-experiments#5]
Hi,
there seems to be an issue with `PyYAML` in one of the docker containers. Executing `./install.py` i get the following error log:

```txt
 > [13/28] RUN pip3 install -r /requirements-pipeline.txt:                                              
0.945 Collecting click==7.1.2 (from -r /requirements-pipeline.txt (line 1))                             
1.156   Downloading https://files.pythonhosted.org/packages/d2/3d/fa76db83bf75c4f8d338c2fd15c8d33fdd7ad23a9b5e57eb6c5de26b430e/click-7.1.2-py2.py3-none-any.whl (82kB)                                          
1.187 Collecting rq (from -r /requirements-pipeline.txt (line 2))                                       
1.272   Downloading https://files.pythonhosted.org/packages/e1/24/5ae8bfcea4f24b32e079da421015ad380eadfc8250892a2470b6f19df0d8/rq-1.15.1-py2.py3-none-any.whl (95kB)
1.305 Collecting PyYAML>=5.3 (from -r /requirements-pipeline.txt (line 3))
1.457   Downloading https://files.pythonhosted.org/packages/cd/e5/af35f7ea75cf72f2cd079c95ee16797de7cd71f29ea7c68ae5ce7be1eda0/PyYAML-6.0.1.tar.gz (125kB)
1.978     Complete output from command python setup.py egg_info:
1.978     running egg_info
1.978     creating pip-egg-info/PyYAML.egg-info
1.978     writing pip-egg-info/PyYAML.egg-info/PKG-INFO
1.978     writing dependency_links to pip-egg-info/PyYAML.egg-info/dependency_links.txt
1.978     writing top-level names to pip-egg-info/PyYAML.egg-info/top_level.txt
1.978     writing manifest file 'pip-egg-info/PyYAML.egg-info/SOURCES.txt'
1.978     Traceback (most recent call last):
1.978       File "<string>", line 1, in <module>
1.978       File "/tmp/pip-build-24wfb92f/PyYAML/setup.py", line 313, in <module>
1.978         python_requires='>=3.6',
1.978       File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 129, in setup
1.978         return distutils.core.setup(**attrs)
1.978       File "/usr/lib/python3.6/distutils/core.py", line 148, in setup
1.978         dist.run_commands()
1.978       File "/usr/lib/python3.6/distutils/dist.py", line 955, in run_commands
1.978         self.run_command(cmd)
1.978       File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
1.978         cmd_obj.run()
1.978       File "/usr/lib/python3/dist-packages/setuptools/command/egg_info.py", line 278, in run
1.978         self.find_sources()
1.978       File "/usr/lib/python3/dist-packages/setuptools/command/egg_info.py", line 293, in find_sources
1.978         mm.run()
1.978       File "/usr/lib/python3/dist-packages/setuptools/command/egg_info.py", line 524, in run
1.978         self.add_defaults()
1.978       File "/usr/lib/python3/dist-packages/setuptools/command/egg_info.py", line 560, in add_defaults
1.978         sdist.add_defaults(self)
1.978       File "/usr/lib/python3/dist-packages/setuptools/command/py36compat.py", line 36, in add_defaults
1.978         self._add_defaults_ext()
1.978       File "/usr/lib/python3/dist-packages/setuptools/command/py36compat.py", line 120, in _add_defaults_ext
1.978         self.filelist.extend(build_ext.get_source_files())
1.978       File "/tmp/pip-build-24wfb92f/PyYAML/setup.py", line 205, in get_source_files
1.978         self.cython_sources(ext.sources, ext)
1.978       File "/usr/lib/python3.6/distutils/cmd.py", line 103, in __getattr__
1.978         raise AttributeError(attr)
1.978     AttributeError: cython_sources
1.978     
1.978     ----------------------------------------
2.001 Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-24wfb92f/PyYAML/
------
dockerfile:24
--------------------
  22 |     user root
  23 |     run pip3 install -r /requirements-fuzzware-eval-docker.txt
  24 | >>> run pip3 install -r /requirements-pipeline.txt
  25 |     run pip3 install -r /requirements-emulator.txt
  26 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c pip3 install -r /requirements-pipeline.txt" did not complete successfully: exit code: 1
Traceback (most recent call last):
  File "/root/hoedur-experiments/./install.py", line 81, in <module>
    main()
  File "/root/hoedur-experiments/./install.py", line 73, in main
    rebuild_docker_containers()
  File "/root/hoedur-experiments/./install.py", line 52, in rebuild_docker_containers
    subprocess.check_call([DIR.joinpath("scripts", "fuzzware", "build_fuzzware_docker.sh")])
  File "/usr/lib/python3.10/subprocess.py", line 369, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '[PosixPath('/root/hoedur-experiments/scripts/fuzzware/build_fuzzware_docker.sh')]' returned non-zero exit status 1.
```

**Comment by Scepticz** [comment]
Hi Ollrogge,

an update of CPython has introduced issues with PyYAML. We have fixed this in the latest version of Fuzzware, but the installation refers to a fixed commit of Fuzzware. I will look into this. Maybe the best solution is to just update the Fuzzware commit ID to the most recent one. This may change results slightly, but should not have a major impact.

Tobi

**Comment by SWW13** [comment]
fixed in 3897405e58ee67bad61e4842eba80e1dbf797c31

**Comment by Ollrogge** [comment]
Thank you ! :)
